src/Api/SecretsManager/Models/Response/SecretResponseModel.cs

```diff
@@ -7,7 +7,7 @@ public class SecretResponseModel : ResponseModel
 {
     private const string _objectName = "secret";
 
-    public SecretResponseModel(Secret secret) : base(_objectName)
+    public SecretResponseModel(Secret secret, bool read, bool write) : base(_objectName)
```

Comment:
We could default to false here.

```suggestion
    public SecretResponseModel(Secret secret, bool read = false, bool write = false) : base(_objectName)
```

So we don't have to write `return new SecretResponseModel(result, false, false);` in multiple places.